deploy/yaml/kmesh.yaml

```diff
@@ -74,7 +74,7 @@ spec:
           command: ["/bin/sh", "-c"]
           args:
             [
-              "./start_kmesh.sh --mode=dual-engine --enable-bypass=false --enable-bpf-log=true --enable-perfmonitor=false",
+              "./start_kmesh.sh --mode=dual-engine --enable-bypass=false --enable-bpf-log=true --profiling=false",
```

Comment:
The config in helm chart should be consistent with this, https://github.com/kmesh-net/kmesh/blob/main/deploy/charts/kmesh-helm/values.yaml#L10